views/forms/self_referral_form.php

```diff
@@ -257,7 +438,7 @@ classAssigned := $18::TEXT,
                                                 while ($enumtype = pg_fetch_assoc($res)) {
                                                     $t = $enumtype ['type'];
                                                     ?>
-                                                    <option value="<?= $t ?>"><?= $t ?></option>
+                                                    <option value="<?= $t ?>" <?php echo (isset($self_pers_state_result) && $self_pers_state_result == $t) ? "selected" : "" ?>><?= $t ?></option>
```

Comment:
There's a lot more of these so I'm not going to comment on all of them, but please change them to the format I've demonstrated above.

Reply:
Updated all fields